lib/runtime/src/instance.rs

```diff
@@ -242,7 +242,24 @@ impl<'a> Iterator for ExportIter<'a> {
 
 // TODO Remove this later, only needed for compilation till emscripten is updated
 impl Instance {
-    pub fn memory_offset_addr(&self, _index: usize, _offset: usize) -> *const usize {
-        unimplemented!("TODO replace this emscripten stub")
+    pub fn memory_offset_addr(&self, index: usize, offset: usize) -> *const u8 {
```

Comment:
Just as a heads up, I'll get rid of this function at some point. Right now, the best replacement is to take `*mut vm::Ctx` as the last argument of the function, get its `local_backing`, and call `.memory(...)` or `.table(...)` on it.

Reply:
Alright. Noted